test/duct/middleware/buddy_test.clj

```diff
@@ -11,52 +12,56 @@
 (defn- auth-identity-handler [{:keys [identity]}]
   {:status 200, :headers {}, :body identity})
 
+(defn- auth-authorization-handler [{:keys [identity] :as request}]
+  (when-not (authenticated? request) (throw-unauthorized))
+  {:status 200, :headers {}, :body identity})
+
 (deftest test-authentication
   (testing "basic authentication"
     (let [middleware (ig/init-key ::buddy/authentication
                                   {:backend :basic
                                    :realm   "Test"
                                    :authfn  test-basic-authfn})
           handler    (middleware auth-identity-handler)]
-      (is (= (handler (-> (mock/request :get "/")
+      (is (= {:status 200, :headers {}, :body "Aladdin"}
+             (handler (-> (mock/request :get "/")
```

Comment:
I changed the order of these test arguments. Since the order should be "expected", "actual". Which makes the test results a bit more clear as well.